### PR TITLE
Update manifest.json for `import_executor`

### PR DESCRIPTION
**Pull request by Anto79-ops**
as per-discussion and suggestion by @bdraco, this will limit/eliminate the time that `dyson_local` blocks the HA event loop.

fixes/solves https://github.com/libdyson-wg/ha-dyson/issues/138

### CODE REVIEW

**Review by dotvezz — APPROVED** [review]

File: custom_components/dyson_local/manifest.json [hunk 1]
```diff
@@ -7,5 +7,6 @@
     "documentation": "https://github.com/libdyson-wg/ha-dyson",
     "iot_class": "local_push",
     "issue_tracker": "https://github.com/libdyson-wg/ha-dyson/issues",
-    "version": "1.3.10"
+    "version": "1.3.10",
+    "import_executor": "true"
```
Comment (dotvezz) [inline custom_components/dyson_local/manifest.json:11]:
```suggestion
    "import_executor": true
```

This changes the type of this value from a string to a boolean, and should help the automated test pass.

File: custom_components/dyson_local/manifest.json [hunk 2]
```diff
@@ -7,5 +7,6 @@
     "documentation": "https://github.com/libdyson-wg/ha-dyson",
     "iot_class": "local_push",
     "issue_tracker": "https://github.com/libdyson-wg/ha-dyson/issues",
```
Comment (bdraco) [inline custom_components/dyson_local/manifest.json:9]:
```suggestion
    "issue_tracker": "https://github.com/libdyson-wg/ha-dyson/issues",
    "import_executor": true,
```

File: custom_components/dyson_local/manifest.json [hunk 3]
```diff
@@ -7,5 +7,6 @@
...
-    "version": "1.3.10"
+    "version": "1.3.10",
+    "import_executor": true
```
Comment (bdraco) [inline custom_components/dyson_local/manifest.json:11]:
```suggestion
    "version": "1.3.10"
```

File: custom_components/dyson_local/manifest.json (hunk 2 above)
Comment (bdraco) [inline custom_components/dyson_local/manifest.json:9]:
```suggestion
    "import_executor": true,
    "issue_tracker": "https://github.com/libdyson-wg/ha-dyson/issues",
```

File: custom_components/dyson_local/manifest.json [hunk 4]
```diff
@@ -7,5 +7,6 @@
     "documentation": "https://github.com/libdyson-wg/ha-dyson",
     "iot_class": "local_push",
```
Comment (bdraco) [inline custom_components/dyson_local/manifest.json:8]:
```suggestion
    "import_executor": true,
    "iot_class": "local_push",
```

**Pull request merged**